drivers/builtin/include/mbedtls/oid.h

```diff
@@ -470,39 +443,6 @@
 extern "C" {
 #endif
 
```

Comment:
A note for potential future improvements: the functions declared here are only used inside TF-PSA-Crypto, except `mbedtls_oid_get_md_alg()` which is also used in X.509 and PKCS7 in Mbed TLS.

Reply:
Actually `mbedtls_oid_get_md_alg()` is not used in crypto! It's only called from `test_suite_md` which validates the OID associated with each hash algorithm. So `mbedtls_oid_get_md_alg()` should move to X.509 as well.